app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -439,25 +439,21 @@ def parse_base_item(item)
     def parse_image_name(image, image_ref)
       parts = %r{
         \A
-        (?:
-          (?<host>.*?)
-          (?::(?<port>.*?))?
-          /(?=.*/)
-        )?
-        (?<name>.*?)
-        (?::(?<tag>[^:]*?))?
+          (?:(?:(?<host>[^\.:\/]+\.[^\.:\/]+)|(?:(?<host2>[^:\/]+)(?::(?<port>\d+))))\/)?
+          (?<name>(?:[^:\/@]+\/)*[^\/:@]+)
+          (?:(?::(?<tag>.+))|(?:\@(?<shatag>sha256:.+)))?
         \z
       }x.match(image)
```

Comment:
The previous regex by @matthewd did not cover localhost:5000/example and tags that appear in this form image@sha256:xxxxxxxxxxxxxxxx
I'm still trying to make it more readable. Any help here would be appreciated.

The regex has to pass the tests in spec/models/ems_refresh/parsers/kubernetes_spec.rb

@simon3z @abonas